cleanlab/object_detection/rank.py

```diff
@@ -369,6 +370,27 @@ def _get_iou(bb1: Dict[str, Any], bb2: Dict[str, Any]) -> float:
     return iou
 
 
+def _has_overlap(label_boxes, labels):
```

Comment:
Can't add dining table case yet since iOU (image_length,image_width) isn't present, this doesn't; affect the thresholding, 
adding image resolution we will need user to pass the parameter of resolutions/ original image paths. 
This can be done in the follow up PR where we'll be considering ratio of areas of boxes and original image as a parameter.

Reply:
ok that's fine, please don't worry about that case